lib/Twig/Extension/DocumentEditableExtension.php

```diff
@@ -43,6 +44,9 @@ public function getFunctions(): array
                 'is_safe' => ['html'],
             ]),
             new TwigFunction('pimcore_iterate_block', [$this, 'getBlockIterator']),
+            new TwigFunction('pimcoreblock_count', [$this, 'getBlockCount'],[
```

Comment:
Why is it called like this and not `pimcore_block_count` or `pimcore_count_block`, which would fit better with the current naming scheme?